presto-main/src/test/java/com/facebook/presto/operator/OperatorAssertion.java

```diff
@@ -34,8 +34,6 @@
 import static com.facebook.presto.type.TypeJsonUtils.appendToBlockBuilder;
 import static com.google.common.base.Preconditions.checkArgument;
 import static io.airlift.testing.Assertions.assertEqualsIgnoreOrder;
-import static org.testng.Assert.assertFalse;
-import static org.testng.Assert.assertTrue;
 
 public final class OperatorAssertion
```

Comment:
Dear Reviewer,

The "Refactor OperatorAssertion.toPages" commit should not be read as a diff. The functions have been rewritten.